ETL/ETL/_handle.h

```diff
@@ -371,21 +372,14 @@ class rhandle : public handle<T>
 	typedef int size_type;
 
 
-	using handle<value_type>::count;
-	using handle<value_type>::unique;
-	using handle<value_type>::operator bool;
-	using handle<value_type>::get;
-	using handle<value_type>::operator*;
-	using handle<value_type>::operator->;
 
-	/*
-	operator const handle<value_type>&()const
-	{ return *this; }
-	*/
+protected:
+#ifdef _DEBUG
+        public:
```

Comment:
Why did you make it public?

Reply:
I just left it the way it was in the handle class.